tools/ide_plugins/pysa-vscode/package.json

```diff
@@ -38,5 +38,34 @@
     "tslint-microsoft-contrib": "^5.0.1",
     "typescript": "^3.5.1",
     "vscode": "^1.1.7"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+      "languages": [{
+          "id": "pysa",
+          "aliases": ["Pysa", "pysa"],
+          "extensions": [".pysa"],
+          "configuration": "./language-configuration.json"
+      },
+      {
+          "id": "taint",
+          "aliases": ["Pysa", "taint"],
+          "extensions": ["taint.config"],
+          "configuration": "./language-configuration.json"
```

Comment:
Should we have a separate language configuration here? JSON has some different rules from Python

Reply:
I've made separate configuration files `model-language-configuration.json` for `.pysa` files and `config-language-configuration.json` for `taint.config` files using the official VSCode configurations for [JSON](https://github.com/microsoft/vscode/blob/main/extensions/json/language-configuration.json) and [Python](https://github.com/microsoft/vscode/blob/main/extensions/python/language-configuration.json)